### PR TITLE
For Peer Review

### PR DESCRIPTION
**Pull request by miKuZune**

### CODE REVIEW

File: Image.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import pygame, sys
+from pygame import *
+
```
Comment (kjwelch1) [inline Image.py:3]: Lack of comments

File: Image.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+pygame.init()
+
+xPos =400
+yPos = 400
+
+WIDTH = 800
+HEIGHT = 600
+BLACK = (0,0,0)
```
Comment (Newtoto) [inline Image.py:11]: Spaces after commas?

File: PCBuilding.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import pygame, sys
+from pygame import *
+
+pygame.init()
+
+WIDTH = 800
+HEIGHT = 600
+BLACK = (0,0,0)
```
Comment (Newtoto) [inline PCBuilding.py:8]: Remove unused variable definitions

File: PCBuilding.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+window = pygame.display.set_mode((WIDTH, HEIGHT),0, 32)
+font = pygame.font.SysFont(None,52)
+text = font.render("",False,BLACK)
+memeTalk = text.get_rect()
+
+
+while True:
+
+    if counterIncrease == False:
```
Comment (Newtoto) [inline PCBuilding.py:30]: You could separate the counter += 1 from the if statement and put it at the top. This way you can have something making "counter = 0" when counter > x, getting rid of the need for the counterIncrease variable.

File: PCBuilding.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import pygame, sys
+from pygame import *
+
+pygame.init()
+
```
Comment (Newtoto) [inline PCBuilding.py:5]: Lacks comments

File: Image.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+    memeTalk.centerx = 400
+    memeTalk.centery = 560
+
+
+    pygame.draw.rect(window, 0, memeTalk, 10)
+    window.blit(text,memeTalk)
+    greyScreen()
+    pygame.display.flip()
+    pygame.display.update()
```
Comment (Newtoto) [inline Image.py:61]: No need for flip and update

File: Image.py [hunk 7]
```diff
@@ -0,0 +1,61 @@
+import pygame, sys
```
Comment (CallumPank) [inline Image.py:1]: These aren't the comments you're looking for.

**Pull request closed without merging**